internal/manager/run.go

```diff
@@ -140,7 +141,14 @@ func Run(
 	}
 
 	setupLog.Info("Initializing Dataplane Client")
-	eventRecorder := mgr.GetEventRecorderFor(KongClientEventRecorderComponentName)
+	var eventRecorder record.EventRecorder
+	if c.EmitTranslationEvents {
+		setupLog.Info("emit translation event enabled, create event recorder for " + KongClientEventRecorderComponentName)
+		eventRecorder = mgr.GetEventRecorderFor(KongClientEventRecorderComponentName)
+	} else {
+		setupLog.Info("emit translation event disabled, discard all events")
```

Comment:
```suggestion
		setupLog.Info("Emit translation event disabled, discard all events")
```
Now we're capitalizing logs